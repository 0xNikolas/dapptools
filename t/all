#!/usr/bin/env bash
set -e -o pipefail
cat <<.; PS4=$'\n$ '; exec 2>&1; set -x
Primary seth(1) test suite
==========================
.

seth accounts
seth status
