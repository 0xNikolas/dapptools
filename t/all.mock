eth_accounts []
=> ["0x841b4d218f211f4e21e856e89d42521b2211201d"]

eth_getBalance ["0x841b4d218f211f4e21e856e89d42521b2211201d"]
=> "0x0"

web3_clientVersion []
=> "Geth/v1.3.5/linux/go1.5.1"

eth_protocolVersion []
=> 63

net_version []
=> 1

eth_gasPrice []
=> "0x4a817c800"

eth_hashrate []
=> "0x0"

net_peerCount []
=> "0xc"

eth_mining []
=> false

net_listening []
=> true

eth_blockNumber []
=> "0x1755d0"

eth_syncing []
=> false
