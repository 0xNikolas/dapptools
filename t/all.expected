Primary seth(1) test suite
==========================

$ seth accounts
0x841b4d218f211f4e21e856e89d42521b2211201d 0x0

$ seth status
block-number    	0x1755d0
client-version  	Geth/v1.3.5/linux/go1.5.1
gas-price       	0x4a817c800
hashrate        	0x0
listening       	true
mining          	false
network-version 	1
peer-count      	0xc
protocol-version	63
sync-current-block	-
sync-highest-block	-
sync-starting-block	-
syncing         	false
